### ISSUE TITLE
Consider changing > prompt in repl area

### ISSUE THREAD

**Issue by blerner** [brownplt/code.pyret.org#269]
The > symbol has meaning in algebra. Consider changing it to » or something else that isn't obviously an operator.

**Comment by schanzer** [comment]
YES PLZ

**Comment by blerner** [comment]
The » character is _slightly_ risky (since some Old-Hand Programmers might think it's a right-shift operator or something).  I'm curious: does the following character display properly on your screens: ⭆  ?  It should look like this preview: ![this preview](http://unicode.scarfboy.com/uci/2b46.png) 
from http://unicode.scarfboy.com/?s=U%2B2B46 and I'm pretty certain that that doesn't have any existing mathematical or programming meaning (except maybe in APL, but we don't need to worry about them...)  If this works on Windows, Linux, Mac, iPad, and a few different browsers, I think we have a safe winner :)

**Comment by blerner** [comment]
Better, since that icon is kinda ugly, and since it's not a standard character with an HTML entity code, we can use ››› (three right single angle quotes), which should work everywhere.

**Comment by shriram** [comment]
FYI, your glyph does *not* render on the browser on my Huawei tablet (which
runs Android crapped over in staggering ways by Huawei).

I think I would rather not use a triple thing-that-looks-like-greater-than.

**Comment by blerner** [comment]
I don't understand, Shriram: I first proposed one glyph, which was a quadruple-lined arrow, and was an obscure Unicode character that I doubted would be rendered well on multiple devices. Then, second, I proposed a standard single html entity (a single right angle quote), that is then repeated three times to produce a triple arrowhead.  Which one isn't visible to you on that Huawei tablet? And, do we care about that tablet? 

(Fwiw, the math teachers I've shown the triple arrowhead to, here, think it's a massive improvement over a single greater-than sign.)
